lib/msal-common/src/network/ThrottlingUtils.ts

```diff
@@ -24,11 +24,11 @@ export class ThrottlingUtils {
     /**
      * Performs necessary throttling checks before a network request.
      * @param cacheManager
-     * @param thumbprint 
+     * @param thumbprint
      */
     static preProcess(cacheManager: CacheManager, thumbprint: RequestThumbprint): void {
         const key = ThrottlingUtils.generateThrottlingStorageKey(thumbprint);
```

Comment:
You can now pass the thumbprint directly to `getThrottlingCache` and have it generate the key

Reply:
Will address this as a separate PR as `removeItem` still needs the key and hence key generation cannot be avoided here.